dataplane/src/main.rs

```diff
@@ -107,34 +95,30 @@ fn start_rte_workers(devices: &Vec<Dev>) {
         let rx_queue = devices[0].rx_queue(RxQueueIndex(i as u16)).unwrap();
         let tx_queue = devices[0].tx_queue(TxQueueIndex(i as u16)).unwrap();
         WorkerThread::launch(lcore_id, move || {
-            let mut pipeline = setup_pipeline();
-            pipeline.start().unwrap();
+            let mut phase0 = DecrementTtl
```

Comment:
We can put this type of sample code into tests for the pipeline, we can stick with just passthrough here.

Reply:
Yes, and also I'm going to drop this whole commit so I won't do that in the PR